initrd/initrd.go

```diff
@@ -177,74 +177,73 @@ func NewFromMapping(workdir, output string, maps ...string) (*InitrdConfig, erro
 					return nil
 				}
 
-				internal = "." + strings.ReplaceAll(filepath.Join(initrdPath, internal), "//", "/")
+				if internal, err = filepath.Rel("/", internal); err != nil {
```

Comment:
```suggestion
				if internal, err = filepath.Rel(string(filepath.Separator), internal); err != nil {
```

Reply:
Or you can leave like that and I modify it in the WIndows PR

Reply:
`string(filepath.Separator)` might not make a difference on Windows because we are generating UNIX file paths here (relative to a literal `/`). For this to work on Windows, the entire path would need to be transformed.